manifests/files.pp

```diff
@@ -33,15 +33,15 @@
 #
 define swap_file::files (
   Enum['absent','present'] $ensure = 'present',
-  Stdlib::Unixpath $swapfile       = '/mnt/swap.1',
+  Stdlib::Absolutepath $swapfile   = '/mnt/swap.1',
   $swapfilesize                    = $facts['memory']['system']['total'],
   Boolean $add_mount               = true,
-  $options                         = 'defaults',
+  String $options                  = 'defaults',
```

Comment:
This is good but we should re-enable the tests which are disabled in puppet-lint.rc or whatever it is... Actually am a bit confused why the modulesync did not fix the file.